_posts/2017-09-28-our-engineering-culture-methodologies-and-philosophies.md

```diff
@@ -0,0 +1,179 @@
+mentorship and coaching as to improving the engineer that requested the pair. It
+further strengthens trust and bonds between individuals as they build a rapport
+and help each other through challenges.
+
+While there are plenty of pro and con articles around the internet about
+pairing, here are some about the positive benefits of it and why we use it
+at Forward Financing:
+- https://raygun.com/blog/how-good-is-pair-programming-really/
+  
+## Testing  
+Testing is another important vehicle that helps reduce technical debt from
+being introduced into new systems and helps vet that requirements are met. Tests
+are a way an engineer can programatically understand if it's functionality is
+in fact doing what is intended.
+
+Here are some articles about why testing is important and why companies decide
+to invest in testing.
+- https://martinfowler.com/bliki/UnitTest.html
+  
+## Continuous Learning  
+Continuous learning is nothing new to businesses, but has made a recent
+resurgence in business culture. This framework allows companies to invest in
+employees by giving them new skills, reinforcing old skills with new twists, and
+more. This has so many positive effects on an organization that it is difficult
+to enumerate them all. Two that I tend to focus on are:
+1) Employee retention
+2) Employee growth.
+
+Here are some links around the internet that describe what continuous learning
+is and why it is important.
+- https://www.td.org/Publications/Magazines/TD/TD-Archive/2017/02/Webex-10-Ways-to-Build-a-Culture-of-Continuous-Learning
+  
+## Management  
+Managing a team of engineers is often likened to herding cats. It is a tall task
+that most people fumble through. Being the liason between senior management and
+engineers also has it's own set of challenges. The two most important pieces
+that any manager can do for engineers are:
+1) Guard rigorously against context switching.
+2) Facilitate growing trust throughout the team.
+
+Beyond those two, there is so much that happens to keep engineers happy,
+motivated, and engaged. Here are some links from individuals that I respect in
+the industry that have gone on to build some amazing engineering cultures:
+- http://firstround.com/review/three-powerful-conversations-managers-must-have-to-develop-their-people/
+- https://hackernoon.com/40-hours-is-enough-86d7166911ea
+- https://m.signalvnoise.com/nobody-hits-the-ground-running-ae4d8fd0e7de
+- https://m.signalvnoise.com/software-has-bugs-this-is-normal-f64761a262ca
+- https://medium.com/startup-patterns/the-dangers-of-measuring-performance-d21cd61426df
+- https://medium.com/javascript-scene/getting-to-10x-results-what-any-developer-can-learn-from-the-best-54b6c296a5ef
+- https://home.opsolutely.com/the-hidden-reason-you-cant-scale-your-engineering-team/
+  
+## Product Development  
+The process around how products are developed are important, but not the end
+all be all. It is important to have structures and processes in place so others
```

Comment:
end-all and be-all

Reply:
processes in place to foster accountability, or something like that.

"others know how to keep others" reads awkward